connectors/src/connectors/confluence/lib/confluence_client.ts

```diff
@@ -244,7 +244,12 @@ export class ConfluenceClient {
         if (retryCount < MAX_RATE_LIMIT_RETRY_COUNT) {
           const delayMs = getRetryAfterDuration(response);
           logger.warn(
-            { endpoint, retryCount, delayMs, headers: response.headers },
+            {
+              endpoint,
+              retryCount,
+              delayMs,
+              headers: Object.fromEntries(response.headers.entries()),
```

Comment:
Can we only dump the keys maybe?

Reply:
We don't know them

Reply:
And we want the values, we'll revert once we have the info